src/molecules/ArticleCard.tsx

```diff
@@ -7,23 +7,18 @@ import theme from '../styles/theme';
 import H from '../atoms/H';
 import P from '../atoms/P';
 import B from '../atoms/B';
+import { Article } from '../utils/types';
 
-interface ArticleProps {
+interface Props {
   reverse?: boolean;
-  title?: string;
-  date?: string;
-  description?: string;
-  category?: string; //Takes: branch, english, olGraver, underhallning, aktuellt or podcast as input
```

Comment:
When I tested the branch I didn't get the static test articles to work because description wasn't a prop anymore. But maybe it works with real articles?

Reply:
It should work with a real data. `description` isn't a property of a "real" article so that's why I removed it!